### ISSUE TITLE
Feature request: better control what gets removed

### ISSUE THREAD

**Issue by webpro** [NullVoxPopuli/package-majors#10]
There's a checkbox to toggle "<1%", which is very helpful.

Yet I think it would be great to "cut" the graph from any version. For instance, when I look at https://majors.nullvoxpopuli.com/q?packages=globby, this currently removes v4 and lower. But let's say v8 or v9 is already old (apparently many folks still stuck on v6, they can't be saved anymore...), and I want to see adoption for v10 and later.

Another example. When I look at my own little package: https://majors.nullvoxpopuli.com/q?packages=knip, this toggle does not remove v0 and v1, so without really understanding why, it would be nice if I can just get rid of those anyway.

**Comment by webpro** [comment]
Oops, just like the issue in #5 this starts to look fine in Chrome v120.

**Comment by NullVoxPopuli** [comment]
> and I want to see adoption for v10 and later.

> it would be nice if I can just get rid of those anyway.

Yeah, there are a number of filters i want to add under some extra ui somewhere for advanced/customized stuff like this.
Plenty of subecosystems have lost causes that we can/should not pay attention to

> does not remove v0 and v1, so without really understanding why

The 1% filter stops removing entries when it hits the lowest version exceeding 1% of all downloads, so v0 exceeds 1%.

**Comment by webpro** [comment]
> > does not remove v0 and v1, so without really understanding why
> 
> The 1% filter stops removing entries when it hits the lowest version exceeding 1% of all downloads, so v0 exceeds 1%.

I get that, but not everyone might, since v1 probably does not, but the label is "hide <1%"
